src/components/Settings/SettingsNewAddressbook.vue

```diff
@@ -47,16 +48,9 @@ export default {
 	directives: {
 		clickOutside
 	},
-	// props: {
-	// 	addressbooks: {
-	// 		type: Array,
-	// 		default() {
-	// 			return {}
-	// 		}
-	// 	}
-	// },
 	data() {
 		return {
+			addressBookRegex: new RegExp('/^[a-zA-Z0-9À-ÿ\s_.!?#|()-]+$/i')
```

Comment:
I'd say, let's drop this for now. We'll see later on :)
(leave it empty we'll fix it later)